statics/js/components/topology.js

```diff
@@ -1031,8 +1031,8 @@ Node.prototype = {
   },
 
   isCaptureAllowed: function() {
-    var allowedTypes = ["device", "veth", "ovsbridge", "geneve", "vlan", "bond", "ovsport",
-                        "internal", "tun", "bridge", "vxlan", "gre", "gretap", "dpdkport"];
+    var allowedTypes = ["device", "veth", "blockdev", "blockdevleaf", "blockdevlvm", "ovsbridge", "geneve",
```

Comment:
I'm not sure you need this

Reply:
looks like things work fine without - so I removed from PR